File: Backend/Stock.Domain/FormOfPayment.cs
using Framework.Domain;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Stock.Domain
{
    public class FormOfPayment : EntityBase
    {
        public string Name { get; set; }

        public int MinNumberOfInstallments { get; set; }

        public int MaxNumberOfInstallments { get; set; }

        public decimal MinimumValue { get; set; }

        public string MoreInformation { get; set; }

        public decimal Rate { get; set; }
    }
}

File: Frontend/src/app/system/_domains/FormOfPayment.ts
import { Injectable } from '@angular/core';
import { _EntityBase } from './_EntityBase';

@Injectable()
export class FormOfPayment extends _EntityBase
{
     public Name: string;
     public MinNumberOfInstallments: number;
     public MaxNumberOfInstallments: number;
     public MinimumValue: number;
     public Rate: number;
     public MoreInformation: string;

}
